scripts/update-compatibility-matrix.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
Can we do this a bit more programmatically instead of search and replacing values?
We could use a README.md.tmpl template file + a config and https://github.com/hairyhenderson/gomplate or a similar tool.